beamer/state_machine.py

```diff
@@ -454,9 +454,9 @@ def _handle_initiate_l1_resolution(
     if _l1_resolution_threshold_reached(claim, context):
         future = context.task_pool.submit(
             run_relayer_for_tx,
-            context.config.l1_rpc_url,
-            context.config.l2b_rpc_url,
-            context.config.l2a_rpc_url,
+            context.config.rpc_urls["l1"],
+            context.fill_manager.web3.provider.endpoint_uri,  # type: ignore
+            context.request_manager.web3.provider.endpoint_uri,  # type: ignore
```

Comment:
Why do we need `type: ignore` here?

Reply:
provider is type BaseProvider which has no endpoint_uri normally. either cast it to HTTPProvider which actually it is or just ignore so i ignored